src/numeric/processes.jl

```diff
@@ -90,7 +90,7 @@ MvNormal(sigma::Float64) = MvNormal(reshape([sigma], 1, 1))
 
 function discretize(mvn::MvNormal)
     n = fill(5, size(mvn.mu))
-    x, w = QE.qnwnorm(n, mvn.mu, mvn.Sigma)
+    x, w = QE.qnwnorm(n, mvn.mu, diagm(vec(mvn.Sigma)))
```

Comment:
Which case was this? Do you have an example where this came up?

If we did get a 2x1 array then it sounds to me like the issue is when we construct the instance of MvNormal